src/client/nav/request.js

```diff
@@ -293,8 +301,10 @@ spf.nav.request.handleChunkFromXHR_ = function(url, options, chunking,
  */
 spf.nav.request.handleCompleteFromXHR_ = function(url, options, timing,
                                                   chunking, xhr) {
-  spf.debug.debug('nav.request.handleCompleteFromXHR_ ',
-                  url, {'extra': chunking.extra, 'complete': xhr.responseText});
+  spf.debug.debug('nav.request.handleCompleteFromXHR_ ', url,
+                  xhr.responseType == 'json' ?
```

Comment:
This line is a bit excessive. Can you determine the correct response object to log on another line and pass it in?